pkg/bootstrap/testdata/all_golden.json

```diff
@@ -338,7 +338,7 @@
               "endpoint": {
                 "address":{
                   "pipe": {
-                    "path": "/tmp/bootstrap/all/SDS"
+                    "path": "./var/run/secrets/workload-spiffe-uds/socket"
```

Comment:
> is there a reason why these file paths are now relatives?

The defined fixed socket path is relative: `WorkloadIdentitySocketPath = "./var/run/secrets/workload-spiffe-uds/socket"`, there are several examples of relatives paths in istio and it was also suggested by one of the maintainers. Also the path in the Envoy bootstrap is relative and thus all these `testadata` files need to have the same relative path otherwise the tests fail.